pycontrails/core/met.py

```diff
@@ -266,7 +268,8 @@ def _preprocess_dims(self, wrap_longitude: bool) -> None:
                 self.data[coord] = arr.astype(COORD_DTYPE)
 
         # Ensure time is np.datetime64[ns]
-        self.data["time"] = self.data["time"].astype("datetime64[ns]", copy=False)
+        if sort:
+            self.data["time"] = self.data["time"].astype("datetime64[ns]", copy=False)
```

Comment:
I don't understand this ... here, we're ensuring the time coordinate has the right dtype, so how is this related to `sort`?

Reply:
You're right, I mixed up the line here. The if-statement is supposed to be two lines below. But yeah let me check if `copy=False` achieves the same, then I can also remove the sort parameter again.